src/include/msvc.h

```diff
@@ -19,8 +19,6 @@
 /*
  * Add MSVC-specific attributes and pragmas to types and function declarations.
  */
-#define	WT_COMPILER_TYPE_ALIGN(x)	__declspec(align(x))
```

Comment:
Have you tested that the new scheme works on Windows? I guess it should, but I'd rather find out before merging.